autogen/agentchat/contrib/teachable_agent.py

```diff
@@ -23,7 +23,7 @@ class TeachableAgent(ConversableAgent):
 
     def __init__(
         self,
-        name="Agent",
+        name="teachable agent",
```

Comment:
we'd better not have space in the name. It will cause an error when being used in a group chat.
BTW, is the teachable agent usable in a group chat? If not, please document that.

Reply:
Ok. Is the main docstr of TeachableAgent the place to document that it hasn't been tested in a group chat?

Reply:
Yes.

Reply:
Done